src/app/download-list/download-list.page.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { NavController } from '@ionic/angular';
+import { AlertService } from '../services/alert/alert.service';
+import { DbService } from '../services/db/db.service';
+
+@Component({
+  selector: 'app-download-list',
+  templateUrl: './download-list.page.html',
+  styleUrls: ['./download-list.page.scss'],
+})
+export class DownloadListPage implements OnInit {
+  constructor(private navCtrl: NavController,private router:Router,private alertService:AlertService,private dbServices:DbService ) { }
+  resultsWithDownload: any[]=[];
```

Comment:
Change it to "downloadsList" or "projectsList"

Reply:
done